examples/annotation_import/pdf.ipynb

```diff
@@ -41,18 +41,21 @@
         "Supported annotations for PDF assets \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ner_with_checklist_subclass_annotation_ndjson = {
</pre><p>Should we include an example of this with Annotation Types?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1031/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Done</p>